cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release_test.go

```diff
@@ -800,6 +629,208 @@ func TestCreateInstalledPackage(t *testing.T) {
 	}
 }
 
+func TestUpdateInstalledPackage(t *testing.T) {
+	testCases := []struct {
+		name               string
+		request            *corev1.UpdateInstalledPackageRequest
+		existingK8sObjs    []testSpecGetInstalledPackages
+		expectedStatusCode codes.Code
+		expectedResponse   *corev1.UpdateInstalledPackageResponse
+		expectedRelease    map[string]interface{}
+	}{
+		{
+			name: "update package (simple)",
+			request: &corev1.UpdateInstalledPackageRequest{
+				InstalledPackageRef: my_redis_ref,
+				PkgVersionReference: &corev1.VersionReference{
+					Version: ">14.4.0",
+				},
+			},
+			existingK8sObjs: []testSpecGetInstalledPackages{
+				redis_existing_spec_completed,
+			},
+			expectedStatusCode: codes.OK,
+			expectedResponse: &corev1.UpdateInstalledPackageResponse{
+				InstalledPackageRef: my_redis_ref,
+			},
+			expectedRelease: flux_helm_release_updated_1,
+		},
+		{
+			name: "returns invalid if installed package doesn't exist",
```

Comment:
s/invalid/not found/ ?

Reply:
yes, indeed. I copied that case from your test, so you might want to correct that one. hehe